transforms/code/repo_level_ordering/ray/src/repo_level_order_transform.py

```diff
@@ -61,23 +51,36 @@
 output_by_langs_key = "output_by_langs"
 output_superrows_key = "combine_rows"
 
-
-@ray.remote
-def add_file(store, group, file_name):
-    store.put(group, file_name)
+group_batch_size = 50
 
 
 class RepoLevelOrderTransform(AbstractTableTransform):
     """
-    Implements a simple copy of a pyarrow Table.
+    Prepares a list of groups in the file and add them to store.
     """
 
     def __init__(self, config: dict[str, Any]):
         """
         Initialize based on the dictionary of configuration information.
-        This is generally called with configuration parsed from the CLI arguments defined
-        by the companion runtime, RepoLevelOrderTransformRuntime.  If running inside the RayMutatingDriver,
-        these will be provided by that class with help from the RayMutatingDriver.
+        The dictionary should contain the following:
+            grouping_column - name of the column to do groupby
+            store_params - A dictionary to create a key-value store which stores key: str -> value: List(str)
+                           The dictionary should contain:
+                           store_type - Type of store. One of [store_type_value_s3, store_type_value_local, store_type_value_ray ].
+                                        store_type_value_ray is preferred for cluster,
+                                        store_type_value_local for single node if cpus are less.
+                           store_backend_dir - A path for
+                                        store types [ store_type_value_s3, store_type_value_local]
+                           s3_creds: A dictionary for S3 creds if using store_type_value_s3.
+                                     {'access_key', 'secret_key', 'url'},
+                                     None if store is ray or local
+                           store_pool_key: A list of actors if store_type_value_ray, otherwise None
+                           store_ray_cpus_key: Num of cpus used, if store_type_value_ray
```

Comment:
i don't see these referenced in the initializer.  You should probably check to be sure they are present there.  This will help keep someone from trying to run the transform outside of ray too.

Reply:
nice to have, but not mandatory.